gym_compete/new_envs/agents/humanoid.py

```diff
@@ -4,7 +4,8 @@
 
 
 def mass_center(mass, xpos):
-    return (np.sum(mass * xpos, 0) / np.sum(mass))[0]
+    # I think the reshape is necessary because np got stricter about broadcasting dims of size 1 vs dims of size 0, if not this calculation might be wrong
```

Comment:
This comment doesn't inspire confidence :) Following line looks correct to me, if you're not sure can you test it a bit?

Reply:
I investigated this, see comment below